rasa_core/policies/ensemble.py

```diff
@@ -55,6 +57,17 @@ def _training_events_from_trackers(training_trackers):
 
         return events_metadata
 
+    def _check_priorities(self) -> None:
+        priorities = []
+        for p in self.policies:
+            if p.priority not in priorities:
+                priorities.append(p.priority)
+            else:
+                logger.warn("Multiple policies with the same priority " +
+                            "found in PolicyEnsemble. When personalizing "
+                            "priorities, be sure to give all policies "
+                            "different priorities.")
```

Comment:
please try to help the user as much as possible: e.g. link to docs, which priority is duplicated, which policy